### PR TITLE
Dashboard UI/UX (II): Aggregate Study Options

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Instead of having too many options on the study/template/service cards, those options that used to pop up new dialogs (Info, Sharing, Classifiers, Quality, Services update, Services boot mode and Save as template) were moved to a "More Options" tabbed dialog.

On the other hand, the request-by-users Rename study option was added to the list of menu options 🙃 

Before:
beforeOptions

After:
afterOptions

### Bonus:
- user_basic_calls.py locust test added
- show user info in e2e
- bug fix: multi-select with filter on to delete shows wrong result
- bug fix: Must click several times Yes to accept confirmation

## Related issue/s
closes https://github.com/ITISFoundation/osparc-simcore/issues/2779
closes https://github.com/ITISFoundation/osparc-simcore/issues/2770
closes https://github.com/ITISFoundation/osparc-simcore/issues/2742

### CODE REVIEW

**Review by pcrespov — COMMENTED** [review]
🎉  improves UX!
- I think that if an item in a menu opens a new window, the name is suffix by ``...``, i.e. ``More Options ...``. Check [this](https://uxdesign.cc/dot-dot-dot-7ce6170bfc7f) out.
- out of curiosity: Is the fill of the star icon in the "quality" tab changing with the score?

**Comment by odeimaiz** [comment]
> 🎉 improves UX!
> 
> * I think that if an item in a menu opens a new window, the name is suffix by `...`, i.e. `More Options ...`. Check [this](https://uxdesign.cc/dot-dot-dot-7ce6170bfc7f) out.
Not a big fun, but we could go for that suffix everywhere across the webapp.

> * out of curiosity: Is the fill of the star icon in the "quality" tab changing with the score?
Nope.

File: services/web/client/source/class/osparc/dashboard/TemplateBrowser.js [hunk 1]
```diff
@@ -216,57 +198,18 @@ qx.Class.define("osparc.dashboard.TemplateBrowser", {
...
+      if (!isCurrentUserOwner) {
         return null;
       }
 
-      const classifiersButton = new qx.ui.menu.Button(this.tr("Classifiers"));
-      classifiersButton.addListener("execute", () => {
-        this.__openClassifiers(studyData);
+      const publishOnPortalButton = new qx.ui.menu.Button(this.tr("Publish on Portal"));
+      publishOnPortalButton.addListener("execute", () => {
+        const msg = this.tr("Not yet implemented");
```
Comment (KZzizzle) [inline services/web/client/source/class/osparc/dashboard/TemplateBrowser.js:209]: ahah nice

**Review by KZzizzle — APPROVED** [review]
awesome! I'm a fan of all the changes!

File: services/web/client/source/class/osparc/dashboard/TemplateBrowser.js (hunk 1 above)
Comment (pcrespov) [reply services/web/client/source/class/osparc/dashboard/TemplateBrowser.js:209]: soon in the best theaters ... 🎉  :-)

**Review by sanderegg — APPROVED** [review]

File: tests/performance/locust_files/director_services.py [hunk 2]
```diff
@@ -17,11 +17,15 @@
 class WebApiUser(FastHttpUser):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.user_id = "my_user_id"
 
     @task()
     def get_services(self):
         self.client.get(
-            f"/v0/services",
+            "v0/services?user_id=" + self.user_id,
```
Comment (pcrespov) [inline tests/performance/locust_files/director_services.py:25]: MINOR: 

```python
 f"v0/services?user_id={self.user_id}" 
```

**Review by pcrespov — APPROVED** [review]

File: tests/performance/locust_files/director_services.py (hunk 2 above)
Comment (odeimaiz) [reply tests/performance/locust_files/director_services.py:25]: 👌

**Pull request merged**